setup.py

```diff
@@ -43,6 +43,11 @@
             "sparsechem@git+ssh://git@gitlab.com/melloddy/wp2/sparsechem.git",
             "protobuf==3.20.*",
         ],
+        "github": [
+            "melloddy_tuner@git+ssh://git@github.com/MELLODDY/MELLODDY-TUNER.git@master",
+            "sparsechem@git+ssh://git@github.com/MELLODDY/SparseChem.git@master",
+            "protobuf==3.20.*",
```

Comment:
Since the repo is public and if there is no more development done on GitLab we could move these to `install_requires` and remove the `gitlab` section entirely, no ?

Reply:
This would be even better if it used git+https so those of behind proxies can use it.
```suggestion
            "melloddy_tuner@git+https://git@github.com/MELLODDY/MELLODDY-TUNER.git@master",
            "sparsechem@git+https://git@github.com/MELLODDY/SparseChem.git@master",
            "protobuf==3.20.*",
```

Reply:
I would not put it in `install requires` as the user may have already installed those repos in his venv, with a different branch for instance, and I guess it would override them